### DISCUSSION TITLE
Error when I upload certain PDF File in RAG

### DISCUSSION THREAD

**Discussion by zouzeTG** [open-webui/open-webui#6860] [General]
[Projet de Constitution_2024_2ème République_VF_241023_145118.pdf](https://github.com/user-attachments/files/17699439/Projet.de.Constitution_2024_2eme.Republique_VF_241023_145118.pdf)
# Bug Report

Failed to upload file (you can test with attached file)

Embedding model : sentence-transformers/all-MiniLM-L6-v2
Reranking Model : BAAI/bge-reranker-v2-m3

## Important Notes

- **Before submitting a bug report**: Please check the Issues or Discussions section to see if a similar issue or feature request has already been posted. It's likely we're already tracking it! If you’re unsure, start a discussion post first. This will help us efficiently focus on improving the project.

- **Collaborate respectfully**: We value a constructive attitude, so please be mindful of your communication. If negativity is part of your approach, our capacity to engage may be limited. We’re here to help if you’re open to learning and communicating positively. Remember, Open WebUI is a volunteer-driven project managed by a single maintainer and supported by contributors who also have full-time jobs. We appreciate your time and ask that you respect ours.

- **Contributing**: If you encounter an issue, we highly encourage you to submit a pull request or fork the project. We actively work to prevent contributor burnout to maintain the quality and continuity of Open WebUI.

- **Bug reproducibility**: If a bug cannot be reproduced with a `:main` or `:dev` Docker setup, or a pip install with Python 3.11, it may require additional help from the community. In such cases, we will move it to the "issues" Discussions section due to our limited resources. We encourage the community to assist with these issues. Remember, it’s not that the issue doesn’t exist; we need your help!

Note: Please remove the notes above when submitting your post. Thank you for your understanding and support!

---

## Installation Method

[Describe the method you used to install the project, e.g., git clone, Docker, pip, etc.]

## Environment

- **Open WebUI Version:** [v0.3.35]
- **Ollama (if applicable):** [V0.4.1]

- **Operating System:** [e.g., Windows 10, macOS Big Sur, Ubuntu 20.04]
- **Browser (if applicable):** [e.g., Chrome 100.0, Firefox 98.0]

**Confirmation:**

- [X ] I have read and followed all the instructions provided in the README.md.
- [X ] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [X ] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

[Describe what you expected to happen.]

## Actual Behavior:

[Describe what actually happened.]

## Description

**Bug Summary:**

I don(t know why certain files in pdf can't be upload (see attached file). 

## Reproduction Details

**Steps to Reproduce:**
[Outline the steps to reproduce the bug. Be as detailed as possible.]

## Logs and Screenshots

\data/uploads/f3fa8810-2e02-4777-b784-68b6b8065b41_Projet de Constitution_2024_2ème République_VF_241023_145118.pdf', 'page': 43}, page_content='')] file-f3fa8810-2e02-4777-b784-68b6b8065b41
Collection file-f3fa8810-2e02-4777-b784-68b6b8065b41 does not exist.
INFO  [open_webui.apps.retrieval.main] adding to collection file-f3fa8810-2e02-4777-b784-68b6b8065b41
INFO:     127.0.0.1:57675 - "GET /ws/socket.io/?EIO=4&transport=polling&t=PCQPYWN&sid=2dfaPmff9tJi1m3bAAAM HTTP/1.1" 200 OK
INFO:     127.0.0.1:57675 - "POST /ws/socket.io/?EIO=4&transport=polling&t=PCQPgE6&sid=2dfaPmff9tJi1m3bAAAM HTTP/1.1" 200 OK
INFO:     127.0.0.1:57675 - "GET /ws/socket.io/?EIO=4&transport=polling&t=PCQPgE7&sid=2dfaPmff9tJi1m3bAAAM HTTP/1.1" 200 OK
INFO:     127.0.0.1:57702 - "GET / HTTP/1.1" 200 OK
INFO:     127.0.0.1:57705 - "GET / HTTP/1.1" 200 OK
INFO:     127.0.0.1:57707 - "GET / HTTP/1.1" 200 OK
INFO:     127.0.0.1:57701 - "GET /api/v1/knowledge/ HTTP/1.1" 200 OK
INFO:     127.0.0.1:57715 - "GET /ws/socket.io/?EIO=4&transport=polling&t=PCQPrgG HTTP/1.1" 200 OK
INFO:     127.0.0.1:57714 - "POST /ws/socket.io/?EIO=4&transport=polling&t=PCQPrQZ&sid=2dfaPmff9tJi1m3bAAAM HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
  + Exception Group Traceback (most recent call last):
  |   File "C:\Python312\Lib\site-packages\starlette\_utils.py", line 87, in collapse_excgroups
  |     yield
  |   File "C:\Python312\Lib\site-packages\starlette\middleware\base.py", line 190, in __call__
  |     async with anyio.create_task_group() as task_group:
  |   File "C:\Python312\Lib\site-packages\anyio\_backends\_asyncio.py", line 680, in __aexit__
  |     raise BaseExceptionGroup(
  | ExceptionGroup: unhandled errors in a TaskGroup (1 sub-exception)

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots/Screen Recordings (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by akshara-tg** [comment]
Even I have the same problem. Any PDF 10 MB does not get uploaded.. The same error I see in the log:
***Collection file-f3fa8810-2e02-4777-b784-68b6b8065b41 does not exist.***

**Comment by Szymok** [comment]
The same here, i tried to downgrade the chromadb to 0.5.5 but nothing
